### PR TITLE
Fixing test_multidut_snappi.

### PR DESCRIPTION
**Pull request by rraghav-cisco**
### Description of PR

Summary:
Minor fixes to test_multidut_snappi.py to make it pass in cisco-8000.

### Type of change

- [X] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [X] 202405

#### What is the motivation for this PR?
There are minor issues in test_multidut_snappi.py that had caused it to fail in cisco-8000. This PR aims to fix these issues.

#### How did you verify the change
```
=========================================================================================================================== PASSES ===========================================================================================================================
 
______________________________________________________________________________________________________________ test_snappi[multidut_port_info0] 
______________________________________________________________________________________________________________
----------------------------------------------------------------------------- generated xml file: /run_logs/ixia/fixed_variables/2024-10-17-22-38-56/tr_2024-10-17-22-38-56.xml --------------------------------------
---------------------------------------- INFO:root:Can not get Allure report URL. Please check logs ---------------------
---------------------------------------------------------------------------------------------- live log sessionfinish ------
------------------------------------------------------------------------------------------------------------- 22:47:39 __init__.pytest_terminal_summary         
L0067 INFO   | Can not get Allure report URL. Please check logs 
================================================================================================================== short test summary info ==========================================================================================
========================= PASSED snappi_tests/test_multidut_snappi.py::test_snappi[multidut_port_info0] ==========================================================================================
=============== 1 passed, 4 warnings in 521.32s (0:08:41) ========================================================================================================== sonic@ixia-sonic-mgmt-whitebox:/data/tests$  
```

### CODE REVIEW

**Comment by rraghav-cisco** [comment]
@selldinesh , @sdszhang : FYI.

**Comment by rraghav-cisco** [comment]
Run result:
`=========================================================================================================================== PASSES ===========================================================================================================================
______________________________________________________________________________________________________________ test_snappi[multidut_port_info0] ______________________________________________________________________________________________________________
----------------------------------------------------------------------------- generated xml file: /run_logs/ixia/fixed_variables/2024-10-17-22-38-56/tr_2024-10-17-22-38-56.xml ------------------------------------------------------------------------------
INFO:root:Can not get Allure report URL. Please check logs
------------------------------------------------------------------------------------------------------------------- live log sessionfinish -------------------------------------------------------------------------------------------------------------------
22:47:39 __init__.pytest_terminal_summary         L0067 INFO   | Can not get Allure report URL. Please check logs
================================================================================================================== short test summary info ===================================================================================================================
PASSED snappi_tests/test_multidut_snappi.py::test_snappi[multidut_port_info0]
========================================================================================================= 1 passed, 4 warnings in 521.32s (0:08:41) ==========================================================================================================
sonic@ixia-sonic-mgmt-whitebox:/data/tests$ 
`

**Comment by wsycqyz** [comment]
@rraghav-cisco Can you put the test result in "How do you verify" section in the PR?

**Comment by wsycqyz** [comment]
The PR test stuck. Try to close the PR and re-open the PR.

File: tests/snappi_tests/test_multidut_snappi.py [hunk 1]
```diff
@@ -25,7 +26,7 @@ def __gen_all_to_all_traffic(testbed_config,
                              prio_dscp_map              # noqa: F811
                              ):
 
-    rate_percent = 100 / (len(port_config_list) - 1)
+    rate_percent = 50 / (len(port_config_list) - 1)
```
[the remark this answers is not in the document]
Comment (rraghav-cisco) [reply tests/snappi_tests/test_multidut_snappi.py:29]: @sdszhang : Done.

[the remark this answers is not in the document]
Comment (rraghav-cisco) [reply tests/snappi_tests/test_multidut_snappi.py:29]: @sdszhang : Any comments?

[the remark this answers is not in the document]
Comment (sdszhang) [reply tests/snappi_tests/test_multidut_snappi.py:29]: Can you add platform check here? I think 50% is only applicable for packet chassis.

[the remark this answers is not in the document]
Comment (rraghav-cisco) [reply tests/snappi_tests/test_multidut_snappi.py]: @sdszhang : Changed the rate to cisco-8000 only.

**Review by sdszhang — APPROVED** [review]
LGTM

**Review by yejianquan — APPROVED** [review]
LGTM

**Pull request merged**